jujugui/static/gui/src/app/init.js

```diff
@@ -318,6 +321,25 @@ class GUIApp {
     }
   }
 
+  /**
+    Sync relation changes to the app relation stores.
```

Comment:
can you add a 'why' to this comment? I don't fully understand it and I know the issue :D